web/ui/dashboard/src/app/private/pages/project/project.component.html

```diff
@@ -1,61 +1,53 @@
-<div class="page layout__sidebar flex padding-left__24px padding-right__24px padding-bottom__40px">
-	<aside class="padding-top__40px border__right position__relative" [ngClass]="{ full: shouldShowFullSideBar }">
-		<button
-			[disabled]="screenWidth < 1150"
-			class="sidebar--button button__clear rounded__50px border__all width__32px height__32px flex flex__align-items-center flex__justify-center position__absolute position--top__8px bg__white"
+<convoy-page [class]="'flex pt-0 pb-0'">
+	<aside class="w-fit pt-10 border-r border-r-grey-10 relative">
+		<convoy-button
+			type="outline"
+			buttonType="button"
+			[class]="'px-0 py-0 border-grey-10 bg-white-100 rounded-100px w-[32px] h-[32px] absolute top-[8px] -right-[16px]'"
+			[disable]="screenWidth < 1150"
 			(click)="shouldShowFullSideBar = !shouldShowFullSideBar"
 		>
-			<svg width="12" height="12" fill="var(--primary-color)" class="open-icon">
+			<svg width="12" height="12" fill="#477DB3" [class]="shouldShowFullSideBar ? 'hidden' : 'block'">
```

Comment:
Please don't enter colour manually like this except it is a one-off usage, see here for fill with a class name https://tailwindcss.com/docs/fill#setting-the-fill-color

Reply:
Where is this `shouldShowFullSideBar` from?